### PR TITLE
Add some repro notes for benchmark failures.

### PR DESCRIPTION
**Pull request by MaheshRavishankar**

### CODE REVIEW

File: benchmarks/README.md [hunk 1]
```diff
@@ -60,3 +60,33 @@ posting results compared against their base commit as comments.
...
+$ python -m pip install iree-tools-tflite-snapshot -f https://github.com/google/iree/releases
+```
+
+2. Confirm the binary `iree-import-tflite` is in your path by running
+```
+$ iree-import-tflite --help
+```
+
+3. Download the TFLite flatbuffer for the failing benchmarks. The
+ocation can be found from [this CMakeLists.txt file](https://github.com/google/iree/blob/f1678fb282d561fbb771d9ebfc97d4a3cdfa3fd3/benchmarks/TFLite/CMakeLists.txt)
```
Comment (ScottTodd) [inline benchmarks/README.md:84]: you can just use a relative path here, without pinning to a specific git commit

```suggestion
3. Download the TFLite flatbuffer for the failing benchmarks. The
location can be found from [this CMakeLists.txt file](./TFLite/CMakeLists.txt)
```

File: benchmarks/README.md [hunk 2]
```diff
@@ -60,3 +60,33 @@ posting results compared against their base commit as comments.
...
+
+4. The input TOSA model can be generated by running
+```
+$ iree-import-tflite <tflite file> -o <tosa output file>
+```
+
+5. The exact flags used to compiler and run the benchmarks can be
+found in [this CMakeLists.txt file](https://github.com/google/iree/blob/f1678fb282d561fbb771d9ebfc97d4a3cdfa3fd3/benchmarks/TFLite/CMakeLists.txt)v
```
Comment (ScottTodd) [inline benchmarks/README.md:92]: same down here

```suggestion
5. The exact flags used to compiler and run the benchmarks can be
found in [this CMakeLists.txt file](./TFLite/CMakeLists.txt)v
```

File: benchmarks/README.md [hunk 3]
```diff
@@ -60,3 +60,33 @@ posting results compared against their base commit as comments.
 ### Other project metrics
 
 TODO(#6161): Collect metrics for miscellaneous IREE system states
+
+## Developer notes
+
+These are ad-hoc notes added for developers to help triage errors.
+
+### Repro of TFLite model errors.
```
Comment (ScottTodd) [inline benchmarks/README.md:68]: nit: remove period from heading

```suggestion
### Repro of TFLite model errors
```

https://developers.google.com/style/periods#periods_with_headings

**Review by ScottTodd — APPROVED** [review]
I'd have a few more documentation style nits if this were aimed at users, but LGTM :P

Thanks for writing this up!

File: benchmarks/README.md [hunk 4]
```diff
@@ -60,3 +60,33 @@ posting results compared against their base commit as comments.
...
+1.  Install `iree-import-tflite`.
+
+```
+$ python -m pip install iree-tools-tflite-snapshot -f https://github.com/google/iree/releases
+```
+
+2. Confirm the binary `iree-import-tflite` is in your path by running
+```
+$ iree-import-tflite --help
+```
```
Comment (GMNGeoffrey) [inline benchmarks/README.md]:
`````suggestion
1.  Install `iree-import-tflite`.

    ```
    $ python -m pip install iree-tools-tflite-snapshot -f https://github.com/google/iree/releases
    ```

2. Confirm the binary `iree-import-tflite` is in your path by running
    ```
    $ iree-import-tflite --help
    ```
`````

File: benchmarks/README.md [hunk 5]
```diff
@@ -60,3 +60,33 @@ posting results compared against their base commit as comments.
...
+
+3. Download the TFLite flatbuffer for the failing benchmarks. The
+ocation can be found from [this CMakeLists.txt file](https://github.com/google/iree/blob/f1678fb282d561fbb771d9ebfc97d4a3cdfa3fd3/benchmarks/TFLite/CMakeLists.txt)
+
+4. The input TOSA model can be generated by running
+```
+$ iree-import-tflite <tflite file> -o <tosa output file>
+```
```
Comment (GMNGeoffrey) [inline benchmarks/README.md:88]:
`````suggestion
4. The input TOSA model can be generated by running
    ```
    $ iree-import-tflite <tflite file> -o <tosa output file>
    ```
`````

**Review by GMNGeoffrey — APPROVED** [review]

**Pull request merged**